torch/_logging/structured.py

```diff
@@ -22,7 +22,7 @@ def intern_string(s: str) -> int:
     return r
 
 
-def from_traceback(tb: Sequence[traceback.FrameSummary]) -> object:
+def from_traceback(tb: Sequence[traceback.FrameSummary]) -> List[Dict[str, Any]]:
```

Comment:
```suggestion
def from_traceback(tb: Sequence[traceback.FrameSummary]) -> List[Dict[str, object]]:
```

Reply:
Interestingly, that triggers this mypy error:

```
  Error (MYPY) [return-value]
    Incompatible return value type (got "list[dict[str, str | int | None]]",
    expected "list[dict[str, object]]")

        34  |                "filename": intern_string(frame.filename),
        35  |            }
        36  |        )
    >>> 37  |    return r

```